common/scaffolding/reports/a11y-3.json

```
@@ -0,0 +1,625 @@
+      "earl:mode": "earl:unknownMode",
+      "@type": [
+        "earl:TestResult",
+        "earl:Assertion"
+      ]
+    },
+    {
+      "earl:test": "WCAG21:status-messages",
+      "earl:subject": "_:subject",
+      "earl:outcome": "earl:inapplicable",
+      "dct:description": "",
+      "earl:mode": "earl:unknownMode",
+      "@type": [
+        "earl:TestResult",
+        "earl:Assertion"
+      ]
+    },
+    {
+      "earl:test": "act:rules/automated/axe_core",
+      "earl:subject": "_:subject",
+      "earl:result": [],
+      "acr:asset": [
+        {
+          "@type": "@json",
+          "@value": "[\n  {\n    \"id\": \"landmark-unique\",\n    \"impact\": \"moderate\",\n    \"tags\": [\n      \"cat.semantics\",\n      \"best-practice\"\n    ],\n    \"help\": \"Ensures landmarks are unique\",\n    \"description\": \"Landmarks should have a unique role or role/label/title (i.e. accessible name) combination\",\n    \"helpUrl\": \"https://dequeuniversity.com/rules/axe/4.2/landmark-unique?application=axeAPI\",\n    \"nodes\": [\n      {\n        \"any\": [\n          {\n            \"id\": \"landmark-is-unique\",\n            \"data\": {\n              \"role\": \"navigation\",\n              \"accessibleText\": null\n            },\n            \"relatedNodes\": [\n              {\n                \"html\": \"<nav class=\\\"gcweb-menu wb-init gcweb-menu-inited\\\" typeof=\\\"SiteNavigationElement\\\" id=\\\"wb-auto-2\\\">\",\n                \"target\": [\n                  \"#wb-auto-2\"\n                ]\n              },\n              {\n                \"html\": \"<nav id=\\\"wb-bc\\\" property=\\\"breadcrumb\\\">\",\n                \"target\": [\n                  \"#wb-bc\"\n                ]\n              },\n              {\n                \"html\": \"<nav>\",\n                \"target\": [\n                  \".gc-main-footer > .container > nav\"\n                ]\n              },\n              {\n                \"html\": \"<nav>\",\n                \"target\": [\n                  \".d-flex > nav\"\n                ]\n              }\n            ],\n            \"impact\": \"moderate\",\n            \"message\": \"The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable\"\n          }\n        ],\n        \"all\": [],\n        \"none\": [],\n        \"impact\": \"moderate\",\n        \"html\": \"<nav>\",\n        \"target\": [\n          \"body > nav\"\n        ],\n        \"failureSummary\": \"Fix any of the following:\\n  The landmark must have a unique aria-label, aria-labelledby, or title to make landmarks distinguishable\"\n      }\n    ]\n  }\n]\n"
+        }
+      ],
+      "earl:mode": "earl:automatic",
+      "acr:severity": "acr:noSeverity",
+      "acr:relevancy": "wbv:acr#noRelevancy",
+      "dct:description": "The landmark error result is a false positive so it passes the axe test.",
+      "earl:outcome": "earl:passed",
+    }
```

The comma need to be removed.

```suggestion
      "earl:outcome": "earl:passed"
    }
```